osconfig_tests/utils/utils.go

```diff
@@ -255,6 +262,7 @@ func CreateComputeInstance(metadataitems []*api.MetadataItems, client daisyCompu
 				Scopes: serviceAccountScopes,
 			},
 		},
+		Labels: map[string]string{"name": name},
```

Comment:
😮After this is fixed, can we have some e2e tests that use labels while others use instance URIs or prefixes? Though maybe probers are a better place for that kind of testing.

Reply:
I think that would be a better added as a prober, we can discuss where we want various test cases later. One we support osInfo filtering then we probably want to add some more e2e tests?